### PR TITLE
Added links to OF-DPA

### LINKED ISSUE #43: OFDPA-related documentation not listed
In [Basebox/OpenFlow](https://docs.bisdn.de/basebox.html#openflow), it is listed that table type specification guidelines are available, but there is no link or additional information on where to find them. Adding such link would make it more clear on these guidelines.

Similarly, a link for OF-DPA could be nice to have here when mentioning "Broadcom OF-DPA".

### PR DESCRIPTION
**Pull request by hilmarm**
* Added links to Broadcom OF-DPA 2.0 specs and manual.
* Mention that it speaks OpenFlow 1.3.4
* Change OFDPA to OF-DPA, as this seems to be the way it is written.
This fixes #43

### CODE REVIEW

**Comment by jklare** [comment]
the line breaks look very strange in the diff, is this intended ?

**Comment by hilmarm** [comment]
> the line breaks look very strange in the diff, is this intended ?

My editor had automatic line breaks for Markdown files. Fixed now.

**Comment by hwoesner** [comment]
Where did you find that OF-DPA speaks OF 1.3.4? I don't think this statement holds.H.Am 08.04.2020 2:19 nachm. schrieb Hilmar Magnusson <notifications@github.com>:Added links to Broadcom OF-DPA 2.0 specs and manual.Mention that it speaks OpenFlow 1.3.4Change OFDPA to OF-DPA, as this seems to be the way it is written.



You can view, comment on, or merge this pull request online at:
  https://github.com/bisdn/bisdn.github.io/pull/50

Commit Summary
added references to OF-DPA, manual and specifications. Also added OpenFlow 3.1.4 which OF-DPA speaks.

File Changes

    M
    basebox.md
    (9)
  

Patch Links:
https://github.com/bisdn/bisdn.github.io/pull/50.patchhttps://github.com/bisdn/bisdn.github.io/pull/50.diff

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.



BISDN GmbHKörnerstraße 7-1010785 BerlinGermanyPhone: +49-30-6108-1-6100Managing Directors: Dr.-Ing. Hagen Woesner, Andreas KöpselCommercial register: Amtsgericht Berlin-Charlottenburg HRB 141569 BVAT ID No: DE283257294

**Comment by hilmarm** [comment]
> Where did you find that OF-DPA speaks OF 1.3.4? I don't think this statement holds.H.Am 

The introduction (chapter 1) of the [OF-DPA 2.0 abstract switch specification](https://github.com/Broadcom-Switch/of-dpa/blob/master/OFDPAS-ETP100-R.pdf) reads:
"...This model is based on the long term stable OpenFlow 1.3.4 specification"

**Comment by jklare** [comment]
looks like there are a lot for strange linebreaks again

**Comment by hilmarm** [comment]
> looks like there are a lot for strange linebreaks again

Fixed.
@hwoesner 
Also reworded OF-DPA, which is now *based* on OpenFlow 1.3.4

File: basebox.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Since baseboxd responds directly to the relevant netlink messages, the intended
 
 ## OpenFlow
 
-baseboxd communicates with switches using the OpenFlow protocol. Our implementation uses the Broadcom OF-DPA flavor specifically. OFDPA table type pattern specification guidelines are available. Switches compatible with Broadcom’s SDK come with the OF Agent. OF Agent is a daemon which serves the OpenFlow connection between the control plane, and the Broadcom-implemented data plane. It enforces the table type pattern specification on the switch side.
+baseboxd communicates with switches using the OpenFlow protocol. Our implementation uses the Broadcom OF-DPA flavor specifically, which is based on [OpenFlow 1.3.4](https://www.opennetworking.org/wp-content/uploads/2014/10/openflow-switch-v1.3.4.pdf). [OF-DPA 2.0 table type pattern specification](https://github.com/Broadcom-Switch/of-dpa/blob/master/OFDPAS-ETP100-R.pdf) guidelines are available. Switches compatible with Broadcom’s SDK come with the OF Agent. OF Agent is a daemon which serves the OpenFlow connection between the control plane, and the Broadcom-implemented data plane. It enforces the table type pattern specification on the switch side.
```
Comment (jklare) [inline basebox.md:90]:
```suggestion
baseboxd communicates with switches using the OpenFlow protocol. Our implementation uses the Broadcom OF-DPA flavor specifically, which is based on [OpenFlow 1.3.4](https://www.opennetworking.org/wp-content/uploads/2014/10/openflow-switch-v1.3.4.pdf). [OF-DPA 2.0 table type pattern specification](https://github.com/Broadcom-Switch/of-dpa/blob/master/OFDPAS-ETP100-R.pdf) guidelines are available. Switches compatible with Broadcom’s SDK come with the OF Agent. OF Agent is a daemon which provides the OpenFlow connection between the control plane, and the Broadcom-implemented data plane. It enforces the table type pattern specification on the switch side.
```

**Comment by rr-santos** [comment]
LGTM now

**Comment by jklare** [comment]
lgtm, thanks

**Pull request merged**